de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/nio/writer/SpanAnnotationException.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.nio.writer;
+
+/**
+ * @author <a href="mailto:shore@ukp.informatik.tu-darmstadt.de">Todd Shore</a>
+ * @since May 2, 2016
+ *
+ */
+public class SpanAnnotationException extends Exception {
```

Comment:
This is not directly related to processing UIMA files: It means that a certain annotation can't be found in a given graph structure (which is on a higher level of abstraction than actual file processing)-- see e.g. usage in `de.tudarmstadt.ukp.dkpro.argumentation.nio.writer.json.JCasTextSpanAnnotationGraphFactory.getAnnotationId(Annotation, Sparse3DObjectMatrix<String, ImmutableSpanTextLabel>, Object2IntMap<ImmutableSpanTextLabel>)`. I tried looking for a UIMA exception which means something similar but couldn't find anything. If you want me to replace it with a certain exception class, let me know.